src/app/(main)/wines/page.tsx

```
@@ -67,6 +72,17 @@ const WinesPage = () => {
           </Button>
         </ModalFooter>
       </Modal>
+      <button onClick={() => open('test02')}>Open Modal 02</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)